test-jest/chart/AreaChart.spec.tsx

```diff
@@ -77,14 +76,48 @@ describe('AreaChart', () => {
       </div>,
     );
 
-    const area = container.querySelector('.recharts-area-area');
+    const mouseEnterEvent = new MouseEvent('mouseover', { bubbles: true, cancelable: true });
+    Object.assign(mouseEnterEvent, { pageX: 200, pageY: 200 });
+    const chart = container.querySelector('.recharts-wrapper');
+    if (!chart) {
+      throw new Error('Chart is null');
+    }
```

Comment:
nit: this can be an assertion instead of throwing an error
`expect(chart)not.toBeNull()`